backend/src/jobs_server/routers/jobs.py

```diff
@@ -104,3 +105,28 @@ async def stop_workload(
             http_status.HTTP_500_INTERNAL_SERVER_ERROR,
             "Failed to terminate workload",
         ) from e
+
+
+@router.get("", response_model_exclude_unset=True)
```

Comment:
This means that I would have to `.get(metadata, None)` from the JSON response on the client side, right?

Reply:
Unless I'm mistaken the Pydantic model takes care of that automatically (by assigning the default, which is `None`).